pkg/systemd/overview-cards/cryptoPolicies.jsx

```diff
@@ -76,41 +80,46 @@ export const CryptoPolicyRow = () => {
                 <CryptoPolicyDialog close={() => setIsOpen(false)}
                                     currentCryptoPolicy={currentCryptoPolicy}
                                     setCurrentCryptoPolicy={setCurrentCryptoPolicy}
+                                    fipsEnabled={fipsEnabled}
                 />
                 }
             </td>
         </tr>
     );
 };
 
+const setPolicy = (policy, setError, setInProgress) => {
+    setInProgress(true);
+
+    const promises = [];
+    if (policy === "FIPS") {
+        promises.push(cockpit.spawn(["fips-mode-setup", "--enable"], { superuser: "require", err: "message" }));
+    } else {
+        promises.push(cockpit.spawn(["fips-mode-setup", "--disable"], { superuser: "require", err: "message" }).then(() =>
+            cockpit.spawn(["update-crypto-policies", "--set", policy], { superuser: "require", err: "message" })));
+    }
+
+    Promise.all(promises).then(() => cockpit.spawn(["shutdown", "--reboot", "now"], { superuser: "require", err: "message" }).catch(error => setError(error)))
+            .catch(error => setError(error))
+            .finally(() => setInProgress(false));
```

Comment:
Sorry, I missed that before -- .then and .catch belong on their own line, and be aligned. Now this has *two* .catch handlers.

Reply:
Aligned, need to figure out why eslint in my editor misindents this :|

Reply:
```
/tmp/source/pkg/systemd/overview-cards/cryptoPolicies.jsx
  103:1  error  Expected indentation of 12 spaces but found 11  indent
  104:1  error  Expected indentation of 12 spaces but found 11  indent
```

GRMBBBL!!

Reply:
Haha -- I told you, you really need to align `.then()` and `.catch` :grin: . But let's not beat this any more now, it's good enough.

Reply:
I meant, put `.then()` on a line of its own.